### PR TITLE
Add nowrap and up-down-left-right directions options for arrowkeys-be…

### PR DESCRIPTION
**Pull request by AdmiralCoco**
…havior

I'm looking to add these as options to the `arrowkeys-behavior` to address some comments from Colin regarding `multi-select` - In particular:
1.  Up/Down key behaviour should also focus previous/next (respectively)
2.  If focus is on the first/last element, we do not want to wrap back to the last/first element

### CODE REVIEW

File: test/focusable-arrowkeys-behavior.html
Comment (AdmiralCoco) [inline test/focusable-arrowkeys-behavior.html]: I added this check here due to `testKeyInteractions` expecting a `focus` event to be fired (which didn't seem to be the case, despite returning the same element [here](https://github.com/Brightspace/d2l-polymer-behaviors-ui/pull/30/files#diff-39de0138258dd6c3cae86c606674984aR182), as opposed to null

I don't think this is the best idea, and I'll write individual tests instead.

**Review by dbatiste — APPROVED** [review]
Looks good!

**Comment by AdmiralCoco** [comment]
Hey @dbatiste, sorry it's been a while but could you to merge this if you're happy with it? (I don't have write access)

**Pull request merged**
